src/modules/thirdParty/sagas.ts

```diff
@@ -167,6 +168,10 @@ export function* thirdPartySaga(builder: BuilderAPI, catalyst: CatalystClient) {
     const queue = new PQueue({ concurrency: MAX_CONCURRENT_REQUESTS })
     const promisesOfItemsBeingUpdated: (() => Promise<ItemCuration>)[] = items.map((item: Item) => {
       const curation = itemCurations.find(itemCuration => itemCuration.itemId === item.id)
+      actionProgressChannel.put({
+        progress: Math.round(((items.length - (queue.size + queue.pending)) / items.length) * 100),
+        tpAction: ThirdPartyAction.PUSH_CHANGES
+      })
```

Comment:
If I'm not wrong, we're not clearing the progress when the modal closes, are we still showing the progress when the process starts until a request is completed?